1.	Identify-Basic-Internet-Problems-with-ping.md

```diff
@@ -0,0 +1,113 @@
+<img src="https://media.licdn.com/mpr/mpr/jc/AAEAAQAAAAAAAAfoAAAAJDRmY2I4ZmE1LTk4ZmQtNGUzOS1hOWQ4LTNkMGJkOWQxNjNkYg.png" width="400"  alt="Sonar screen">
+
+## Identify Basic Internet Problems with "ping"
```

Comment:
This particular first heading must be either heading 1 or heading 2
i.e.
```md
# Identify Basic Internet Problems with "ping"
or
### Identify Basic Internet Problems with "ping"
```

----
**Reason:** because of how our beloved @camperbot extracts wiki to show in chat